### ISSUE TITLE
Entire scrollable region does not get keyboard focus

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7212]
### Where To Find Bug
`/projects`

### Expected Behaviour
The "Instructions" and "Notes and Credit" scrollable regions should get keyboard focus and an audio description.

### Actual Behaviour
The screen reader announces links that are inside of the scrollable region but does not announce the region itself to the AT user.

### Steps To Reproduce

- Open https://scratch.mit.edu/ in any browser on your computer
- Turn-on your screen reader of choice
- Click on any project
- Tab through the `projects` page until keyboard focus lands on the "Instructions" or "Notes and Credit" scrollable region
- Notice the none of these regions get keyboard focus, but focus lands on links inside of these regions

### System Details
macOs Ventura, Chrome Browser, VoiceOver (AT)

**Comment by MaterArc** [comment]
Yep
